### PR TITLE
fix some tests relative to vaddr

### PR DESCRIPTION
**Pull request by ret2libc**

### CODE REVIEW

File: t.formats/elf/avr [hunk 1]
```diff
@@ -5,11 +5,12 @@
 for a in . .. ../.. ; do [ -e $a/tests.sh ] && . $a/tests.sh ; done
 
 NAME='avr entrypoint'
+BROKEN=1
 FILE=../../bins/elf/bugurtos-avr.elf
 ARGS=
 CMDS='ie~addr
 '
-EXPECT='vaddr=0x00000074 paddr=0x00000074 baddr=0x00000000 laddr=0x00000000
+EXPECT='vaddr=0x00000000 paddr=0x00000074 baddr=0x00000000 laddr=0x00000000
```
Comment (ret2libc) [inline t.formats/elf/avr]: output of readily
```
  Version:                           0x1
  Entry point address:               0x0
  Start of program headers:          52 (bytes into file)
....
  Type           Offset   VirtAddr   PhysAddr   FileSiz MemSiz  Flg Align
  LOAD           0x000074 0x00000000 0x00000000 0x0234c 0x0234c R E 0x1
```

File: t.formats/elf/avr [hunk 2]
```diff
@@ -19,19 +20,19 @@ FILE=../../bins/elf/bugurtos-avr.elf
 ARGS=
 CMDS='?v sym.main
 '
-EXPECT='0x184
+EXPECT='0x308
```
Comment (ret2libc) [inline t.formats/elf/avr]: output of readily
```
...
   153: 00800100     0 NOTYPE  GLOBAL DEFAULT    2 __bss_start
   154: 00000308   680 FUNC    GLOBAL DEFAULT    1 main
   155: 000000ca     0 NOTYPE  WEAK   DEFAULT    1 __vector_4
...
```

File: t.formats/mach0/thumb [hunk 3]
```diff
@@ -19,7 +19,7 @@ FILE=../../bins/mach0/thumb-test
 ARGS=
 CMDS='ie~vaddr
 '
-EXPECT='vaddr=0x0000bfa4 paddr=0x00007fa5 baddr=0x00003fff laddr=0x00000000
+EXPECT='vaddr=0x0000bfa4 paddr=0x00007fa5 baddr=0x00004000 laddr=0x00000000
```
Comment (ret2libc) [inline t.formats/mach0/thumb]: output of otool -l
```
...
Load command 1
      cmd LC_SEGMENT
  cmdsize 260
  segname __TEXT
   vmaddr 0x00004000
   vmsize 0x00008000
  fileoff 0
 filesize 32768
...
```

File: t.formats/elf/helloworld-gcc-elf [hunk 4]
```diff
@@ -39,8 +39,8 @@ EXPECT='0x080484b0 string[13] "Hello_world_"
 0x08049694 CCu "[22] va=0x08049694 pa=0x00000694 sz=24 vsz=24 rwx=-rw- .got.plt"
 0x080496ac CCu "[23] va=0x080496ac pa=0x000006ac sz=8 vsz=8 rwx=-rw- .data"
 0x080496b4 CCu "[24] va=0x080496b4 pa=0x000006b4 sz=4 vsz=4 rwx=-rw- .bss"
-0x08048000 CCu "[31] va=0x08048000 pa=0x00000000 sz=52 vsz=52 rwx=-rw- ehdr"
```
Comment (ret2libc) [inline t.formats/elf/helloworld-gcc-elf]: seems like my new patch at https://github.com/radare/radare2/pull/3127 change the orders of these things. Is it a problem?

Comment (radare) [inline t.formats/elf/helloworld-gcc-elf]: order of sections?

Comment (ret2libc) [inline t.formats/elf/helloworld-gcc-elf]: no it's only the order of comments i think.

Comment (radare) [inline t.formats/elf/helloworld-gcc-elf]: nah, no issue at all for this

**Comment by radare** [comment]
lgtm

**Comment by ret2libc** [comment]
Ok, but let's wait until we will merge https://github.com/radare/radare2/pull/3127

**Comment by radare** [comment]
hum, but how do you want travis to test this if its not merged?

**Comment by ret2libc** [comment]
We will merge it, but just before merging the other PR. Because if you merge these tests now, some of them will be broken.

**Comment by radare** [comment]
you can mark them as broken too, and get some [FIX] in your branch

**Pull request closed without merging**